src/EventListener/SchemaFilterListener.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Bundle\MigrationsBundle\EventListener;
+
+use Doctrine\DBAL\Schema\AbstractAsset;
+use Doctrine\Migrations\Tools\Console\Command\DoctrineCommand;
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+
+/**
+ * Acts as a schema filter able that hides the migration metadata table except
```

Comment:
Sounds weird. Did you mean something like this?
```suggestion
 * Acts as a schema filterable that hides the migration metadata table except
```

Reply:
I think I changed my mind about how to phrase this mid-sentence, I'll rephrase.

Reply:
That's what happens to me all the time :smile: